ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/Constants.java

```diff
@@ -48,7 +48,9 @@
   public static int MIN_SEED_LOOKAHEAD = 1; // 2^0 epochs (6.4 minutes)
   public static int ACTIVATION_EXIT_DELAY = 4; // 2^2 epochs (25.6 minutes)
   public static int EPOCHS_PER_ETH1_VOTING_PERIOD = 16; // 2^4 epochs (~1.7 hours)
+  public static int SLOTS_PER_HISTORICAL_ROOT = 8192; // 2^4 epochs (~1.7 hours)
   public static int MIN_VALIDATOR_WITHDRAWABILITY_DELAY = 256; // 2^8 epochs (~27 hours)
+  public static int PERSISTENT_COMMITTEE_PERIOD = 2048; // 2^8 epochs (~27 hours)
```

Comment:
Small change to comment.
```suggestion
  public static int PERSISTENT_COMMITTEE_PERIOD = 2048; // 2^11 epochs (~9 days)
```